test/known_issues/test-http-path-contains-unicode.js

```diff
@@ -10,10 +10,10 @@ const http = require('http');
 
 const expected = '/café🐶';
 
+//Sanity check for café🐶 is café🐶
 assert.strictEqual(
   expected,
-  '/caf\u{e9}\u{1f436}',
-  'Sanity check that string literal produced the expected string'
+  '/caf\u{e9}\u{1f436}'
```

Comment:
Nit: can you please swap the arguments (actual first expected second) and put everything on the same line now that it doesn't take more than 80 chars? Thank you.